aptos-move/framework/aptos-framework/sources/account/account.move

```diff
@@ -1132,14 +1133,14 @@ module aptos_framework::account {
 
     #[test_only]
     public fun set_signer_capability_offer(offerer: address, receiver: address) acquires Account {
-        let account_resource = borrow_global_mut<Account>(offerer);
-        option::swap_or_fill(&mut account_resource.signer_capability_offer.for, receiver);
+        let account_resource = &mut Account[offerer];
+        &mut account_resource.signer_capability_offer.for.swap_or_fill(receiver);
```

Comment:
can be reduced, also why is there an &mut on the second line?

Reply:
Updated